roles/oidcng/tasks/main.yml

```diff
@@ -63,6 +63,13 @@
   notify:
     - "restart oidcng"
 
+- name: copy oidcng well-known discovery configuration
+  template:
+    src: "openid-configuration.json.j2"
+    dest: "{{ oidcng_dir }}/openid-configuration.json"
+  notify:
+    - "restart oidcng"
```

Comment:
Do we need to restart or would a reload be sufficient?

Reply:
Restart and reload are the same for a Spring boot application